rpcserver.go

```diff
@@ -64,6 +64,11 @@ const (
 	// permitted.
 	maxLtcPaymentMSat = lnwire.MilliSatoshi(math.MaxUint32) *
 		btcToLtcConversionRate
+
+	// defaultPayAttemptTimeout is a duration that we'll use to determine
+	// if we should give up on a payment attempt. This will be used if a
+	// value isn't specified in the request.
+	defaultPayAttemptTimeout = time.Duration(time.Second * 60)
```

Comment:
would it make more sense to define this in `routing.DefaultPayAttemptTimeout`?

Reply:
Basically just exporting the constant that was already there.

Reply:
Yes, I think that is better. fixed.